doc/nrf/dm_code_base.rst

```diff
@@ -72,7 +72,7 @@ A :ref:`manifest repository <zephyr:west-manifests>`, `sdk-nrf`_, contains a fil
 The |NCS| repository structure has a star topology, with the `sdk-nrf`_ repository being the center of the star and all other repositories being west projects that are managed by :file:`west.yml`.
 This is equivalent to topology T2 in the :ref:`west documentation <zephyr:west-basics>`.
 
-.. figure:: images/ncs-west-repos.png
+.. figure:: images/ncs-west-repos.svg
```

Comment:
Asset Tracker V2 - Do you know which states we should highlight? I did not create this so I'm not sure what the original intent was behind the colors.

ot-arch_2x.svg - Sure, that's a good suggestion. I will revert the image and add the caption.

pelion_client_states.svg - My thinking in changing the colors was to show a progression/change in states to guide the user's eyes through the diagram since it seemed a bit flat to use just one tone of blue. I can see how using the dark blue emphasizes the Suspended state too much though. What about using the dark blue for the Registered state and the lighter tones for the other states? I can update the color legend too.

ncs-west-repos.svg - Sure, I will add in another block and description for "Other modules".

Reply:
For Asset Tracker, I think your contact is @peknis . 
For perlion client states, I'm fine with lighter tones, unless @pdunaj disagrees.
Also, @emob-nordic , can you have a look at the CAF state diagrams?

Reply:
To get the right info about the Asset Tracker images from the SME, that would be perhaps @coderbyheart.

Reply:
All images in [asset_tracker_v2](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/README.html) were made by @simensrostad.